qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -355,21 +306,9 @@ def reset_error(prob0, prob1=0):
     if prob0 < 0 or prob1 < 0 or prob0 > 1 or prob1 > 1:
         raise NoiseError("Invalid reset probabilities.")
     noise_ops = [
-        ([{
-            'name': 'id',
-            'qubits': [0]
-        }], 1 - prob0 - prob1),
-        ([{
-            'name': 'reset',
-            'qubits': [0]
-        }], prob0),
-        ([{
-            'name': 'reset',
-            'qubits': [0]
-        }, {
-            'name': 'x',
-            'qubits': [0]
-        }], prob1),
+        ([(IGate(), [0])], 1 - prob0 - prob1),
+        ([(Reset(), [0])], prob0),
+        ([(Reset(), [0]), (XGate(), [0])], prob1)
```

Comment:
I know this isn't actually changed code, but is there a problem here with the identity probability?  Looks to me like `prob0 = prob1 = 0.7` would pass the validity checks, but give a negative probability on one operator.